core/src/main/java/com/scalar/db/storage/common/checker/DynamoOperationChecker.java

```diff
@@ -0,0 +1,26 @@
+package com.scalar.db.storage.common.checker;
+
+import com.scalar.db.api.Put;
+import com.scalar.db.api.TableMetadata;
+import com.scalar.db.common.TableMetadataManager;
+
+public class DynamoOperationChecker extends OperationChecker {
```

Comment:
I think we should put `DynamoOperationChecker` in the `com.scalar.db.storage.dynamo` package. What do you think?

Reply:
Yes, that makes more sense. Thank you